### DISCUSSION TITLE
Monitor Electric Vehicule charger SE inverter

### DISCUSSION THREAD

**Discussion by FranciscoBigWings** [WillCodeForCats/solaredge-modbus-multi#115] [General]
Hello.

One of my SE inverter is a EV charger and I wonder if we could read the power sent to the car as an independent load, this way we could read how much energy we are "using and storing" in the car.

Would this be possible? How could I help make it possible?

**Comment by WillCodeForCats** [comment]
I'm not aware of any registers for that (or other) devices, so no, not through this integration.

I would recommend getting a separate branch circuit monitor for that data.

Once you have a circuit monitor (there are many types, I don't know all the options for all countries) you can add it to the energy dashboard under "individual devices".
